src/windows/track_list.c

```diff
@@ -181,7 +181,12 @@ static void window_track_list_close(rct_window *w)
 static void window_track_list_select(rct_window *w, int index)
 {
 	w->track_list.var_480 = index;
-
+	if (_loadedTrackDesign == NULL) // Displays a message if the ride can't load, fix #4080
+		{
```

Comment:
Youve indented this one too many times it should be one tab for the bracket and two tab for the window_error_open line.